view/frontend/web/internals/autocomplete.css

```diff
@@ -398,9 +398,19 @@
     right: 9px;
 }
 
+#algolia-searchbox .magnifying-glass {
+    background: url("data:image/svg+xml;utf8,<svg width=\'40\' height=\'40\' viewBox=\'0 0 40 40\'  fill=\'%23A6A6A6\' xmlns=\'http://www.w3.org/2000/svg\'><path d=\'M15.553 31.107c8.59 0 15.554-6.964 15.554-15.554S24.143 0 15.553 0 0 6.964 0 15.553c0 8.59 6.964 15.554 15.553 15.554zm0-3.888c6.443 0 11.666-5.225 11.666-11.668 0-6.442-5.225-11.665-11.668-11.665-6.442 0-11.665 5.223-11.665 11.665 0 6.443 5.223 11.666 11.665 11.666zm12.21 3.84a2.005 2.005 0 0 1 .002-2.833l.463-.463a2.008 2.008 0 0 1 2.833-.003l8.17 8.168c.78.78.78 2.05-.004 2.833l-.462.463a2.008 2.008 0 0 1-2.834.004l-8.168-8.17z\' fill-rule=\'evenodd\'/></svg>") no-repeat center right / 20px;
+    border: none;
+    bottom: 7px;
+    height: 20px;
+    position: absolute;
+    right: 7px;
+    width: 20px;
+}
```

Comment:
To prevent the button styling, I'd remove the box shadowing and browser default outline stylings. WDYT? 

```suggestion
    right: 7px;
    width: 20px;
    box-shadow: none;
    outline: none;
}
```

Reply:
I'd added `box-shadow` in eca5ca397c5488f5e0e4a9e75b4cd3f5a8c31a45, but omitted from 248b8a60bafdc8668839e7a7c740252cf00aa241 because it didn't seem to be necessary in my testing. I have tested both Firefox and Chrome on an up-to-date `2.4-develop` branch; what Magento version(s) are you using and which browser(s)?

Reply:
@bsuravech Please can you provide some information about how to reproduce the bug you've described?

Reply:
@bsuravech What can I do to progress this?

Reply:
Hi @fredden, just to be sure, I'd like to add what I suggested. As you can see in my screenshot on my local environment (MacOS Chrome):

![Screenshot 2021-01-26 at 11 26 18](https://user-images.githubusercontent.com/7048136/105833674-d44fd900-5fc9-11eb-97a1-5115bfd5d540.png)

Reply:
Thanks for the update. What version of Magento are you testing with?

Reply:
Magento 2  v2.3.5. Shouldn't be an issue with adding that style unless you see it effecting your changes.

Reply:
Thanks for confirming. I've tested with Magento version 2.3.6-p1 (and 2.3.5) now and can see what you're seeing. This isn't a problem on v2.4.x. I'll update the styles accordingly.